packages/backend/server/src/__tests__/e2e/indexer/aggregate.spec.ts

```diff
@@ -0,0 +1,237 @@
+import { randomUUID } from 'node:crypto';
+
+import { indexerAggregateQuery, SearchTable } from '@affine/graphql';
+
+import { IndexerService } from '../../../plugins/indexer/service';
+import { Mockers } from '../../mocks';
+import { app, e2e } from '../test';
+
+const isElasticsearch =
```

Comment:
should not use different provider in tests and assert different output.
either separate them to different files and skip tests if provide not match, or simply use one of them for all the tests